packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ImageAndTextViewer/ImageAndTextViewer.stories.js

```diff
@@ -43,10 +42,12 @@ export function Default ({ dark }) {
       themeMode={dark ? 'dark' : 'light'}
     >
       <Provider classifierStore={store}>
-        <Box height='800px' width='large'>
-          <ImageAndTextViewerConnector
-            loadingState={asyncStates.success}
+        <Box width='large'>
+          <ImageAndTextViewer
             subject={store.subjects.active}
+            loadingState={store.subjectViewer.loadingState}
+            onError={store.subjectViewer.onError}
+            onReady={store.subjectViewer.onSubjectReady}
```

Comment:
Are these being added by hand because the story isn't wrapped in a `SubjectViewer`?

Reply:
Yes. I don't think any of the specific subject viewer stories use `SubjectViewer`, but maybe they should. I'll make a note of it in a different PR I'm working on adding back stories with ImageToolbar.

Reply:
The method here allows you to set the loading state from Storybook controls, which might be useful.